kivy/uix/button.py

```diff
@@ -12,6 +12,9 @@
 
     button = Button(text='Hello world', font_size=14)
 
+.. note:: To fit the Button to its :attr:`~Button.text`, bind size to :attr:`~kivy.uix.label.Label.texture_size`
```

Comment:
I don't think this should be here, as we cannot document about parent behavior in every subclass. At some point we have to assume that the user will read the parent doc, rather than duplicate it, or mention it again. Especially, since it's mentioned right above that, that it inherits from Label.